custom_components/petlibro/button.py

```diff
@@ -0,0 +1,74 @@
+"""Support for PETLIBRO buttons."""
+
+from __future__ import annotations
+
+from collections.abc import Callable, Coroutine
+from dataclasses import dataclass
+from functools import cached_property
+from typing import Any, Generic
+
+from homeassistant.components.button import ButtonEntity, ButtonEntityDescription
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import PetLibroHubConfigEntry
+from .entity import PetLibroEntity, _DeviceT, PetLibroEntityDescription
+from .devices.device import Device
+from .devices.feeders.feeder import Feeder
+
```

Comment:
Unused import and move `logging`

```suggestion
from __future__ import annotations

from collections.abc import Callable, Coroutine
from dataclasses import dataclass
from typing import Any, Generic
from logging import getLogger

from homeassistant.components.button import ButtonEntity, ButtonEntityDescription
from homeassistant.const import EntityCategory
from homeassistant.core import HomeAssistant
from homeassistant.helpers.entity_platform import AddEntitiesCallback

from . import PetLibroHubConfigEntry
from .entity import PetLibroEntity, _DeviceT, PetLibroEntityDescription
from .devices.device import Device
from .devices.feeders.feeder import Feeder


_LOGGER = getLogger(__name__)


```